src/main/java/com/yurjinia/project_structure/comment/entity/CommentEntity.java

```diff
@@ -12,14 +12,16 @@
 import jakarta.persistence.PrePersist;
 import jakarta.persistence.PreUpdate;
 import jakarta.persistence.Table;
-import lombok.Getter;
-import lombok.Setter;
+import lombok.*;
 
 import java.time.LocalDateTime;
 
 @Entity
 @Getter
 @Setter
+@Builder
```

Comment:
recheck the annotations and remove redundant ones

Reply:
I'll check it in CommentEntity when I make it